keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -61,18 +61,23 @@ open class SqlResourceRepository(
     }
   }
 
-  override fun get(id: String): Resource<ResourceSpec> {
-    return sqlRetry.withRetry(READ) {
+  override fun get(id: String): Resource<ResourceSpec> =
+    readResource(id, this::parseAndMigrateResource)
+
+  override fun getRaw(id: String): Resource<ResourceSpec> =
```

Comment:
I find the use of the word "raw" here a bit misleading, since the return type is the same as the "non-raw" version. Perhaps `getOriginalResource` here and `getCanonicalResource` for the migrated version?